.codeclimate.yml

```diff
@@ -1,42 +1,26 @@
 ---
-engines:
+version: "2"
+plugins:
   shellcheck:
     enabled: true
-  checkstyle:
-    enabled: true
-    channel: "beta"
   csslint:
     enabled: true
   duplication:
     enabled: true
     config:
-      enabled: true
-      threshold: 7
       languages:
-        - typescript
-        - javascript
-        - python
-        - php
+        - typescript:
+        - javascript:
+        - python:
+        - php:
   eslint:
     enabled: true
```

Comment:
We need to add:
```yml
channel: "eslint-8"
```
See [docs](https://docs.codeclimate.com/docs/eslint#eslint-versions)